src/components/dropdown-filter/notes.md

```diff
@@ -0,0 +1,14 @@
+#Designer Notes
```

Comment:
Error in MD here

![image](https://user-images.githubusercontent.com/26925206/53957349-073b0c00-40d6-11e9-80ef-7078a30ebdfb.png)